packages/manager/modules/nutanix/src/dashboard/nodes/list/translations/Messages_fr_FR.json

```
@@ -10,5 +10,7 @@
   "nutanix_dashboard_nodes_list_data_hidden": "CACHÉ",
   "nutanix_dashboard_nodes_list_add_nodes": "Ajouter des noeuds",
   "nutanix_dashboard_nodes_add_node_max_node_tooltip": "Vous avez atteint le nombre maximum de noeuds autorisés dans un cluster",
-  "nutanix_dashboard_nodes_suspended_tooltip": "Pour ré-activer le noeud veuillez contacter le support"
+  "nutanix_dashboard_nodes_suspended_tooltip": "Pour ré-activer le noeud veuillez contacter le support",
```

To be confirmed but I think a comma is necessary here
```suggestion
  "nutanix_dashboard_nodes_suspended_tooltip": "Pour ré-activer le noeud, veuillez contacter le support",
```